Makefile

```diff
@@ -0,0 +1,33 @@
+export KUBE_PROMETHEUS_VERSION ?= 0.23.2
+export KUBE_PROMETHEUS_BASE_URL ?= https://raw.githubusercontent.com/coreos/prometheus-operator/v$(KUBE_PROMETHEUS_VERSION)/helm/grafana/dashboards/
+
+KUBE_PROMETHEUS_DASHBOARDS :=
+KUBE_PROMETHEUS_DASHBOARDS += deployment-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += kubernetes-capacity-planning-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += kubernetes-cluster-health-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += kubernetes-cluster-status-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += kubernetes-control-plane-status-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += kubernetes-resource-requests-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += nodes-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += pods-dashboard.json
+KUBE_PROMETHEUS_DASHBOARDS += statefulset-dashboard.json
+
+export DS_PROMETHEUS ?= Prometheus
+
+apply: kube-prometheus/apply nginx/apply
```

Comment:
I think `import` would be a more apropos target name as we are importing and then transforming (but not loading) the dashboards.